scss/civicrm/contact/pages/_memberships.scss

```diff
@@ -522,17 +532,37 @@
         }
       }
 
-      tr td.label {
-        background: none !important;
-        padding: 10px 30px !important;
+      tr {
+        th {
+          padding: 10px 20px !important;
```

Comment:
can't we avoid adding `!important` ? is there no way to prioritize these rules without using `!important` ?

Reply:
Nope! it is not working without it, due to many already previous !important s.

Reply:
Fixed